src/main/java/com/revature/controllers/PersonController.java

```diff
@@ -24,99 +27,102 @@
 public class PersonController {
 	
 	@Autowired
-	private PersonLogic personLogic;
-	
-	@Autowired
-	private PersonRoleLogic personRoleLogic;
+	private PersonLogic personLogic;	
 	
-	@Autowired
-	private BatchLogic batchLogic;
+	@RequestMapping(method = RequestMethod.GET, value = "persons/{id}")
+		public ResponseEntity<Person> getPersonById(@PathVariable("id") Integer id ){
+
+			Person person = personLogic.getPersonById(id);
 		
+			
+			if (person != null) {
+				
+				return ResponseEntity.ok(person);
+				
+			} else {
+				
+				System.out.println("ERROR!");				
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS106)